core/templates/pages/contributor-dashboard-page/opportunities-list-item/opportunities-list-item.component.ts

```diff
@@ -91,6 +92,10 @@ export class OpportunitiesListItemComponent {
           this.inReviewProgressStyle = {
             width: inReviewTranslationsPercentage + '%'
           };
+          this.opportunityButtonDisabled = (
+            this.opportunity.translationsCount +
+            this.opportunity.inReviewCount >=
```

Comment:
When would translationsCount + inReviewCount be > totalCount?

Reply:
Theoritically it should be translationsCount + inReviewCount be = totalCount once all the cards are translated or in review? I changed it accordingly.